client/templates/nav/nav.js

```diff
@@ -40,15 +40,11 @@ Template.notificationsDropdown.onRendered(function() {
 
 Template.notificationsDropdown.helpers({
   hasNotifications: function(){
-    if(Meteor.user() && Meteor.user().notificationsCount() > 0){
-      return true;
-    }
-
-    return false;
+    return Meteor.user() && Meteor.user().notificationsCount() > 0;
```

Comment:
Currently it's still part of the user - It just queries the Notifications collection instead of itself - so it's a helper (nothing is actually stored on the User related to Notifications, just the helper method is attached).